purchase_request/models/purchase_request.py

```diff
@@ -8,8 +8,9 @@
     ("draft", "Draft"),
     ("to_approve", "To be approved"),
     ("approved", "Approved"),
-    ("rejected", "Rejected"),
+    ("in_progress", "In progress"),
```

Comment:
Non-blocking. This state should come after the draft, isn't it?

Reply:
Not really. The new 'In Progress' state is designed to help the person managing purchases track the progress of purchase requests, specifically the final step, which is purchasing the products on approved requests.

That's why they are automatically set to 'In Progress' when an RFQ is created, indicating that the purchasing process has begun. This ensures that the person in charge knows that action has been taken on the purchase request, in contrast to those in the 'Approved' state, where the purchasing process is still pending.